2024-03-13 20:15:35,444 - regression_class_books_logistic - INFO - Running FSS
2024-03-13 20:15:35,614 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:15:35,614 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:15:35,617 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:15:35,617 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:15:35,619 - regression_class_books_logistic - INFO - Model 1
2024-03-13 20:15:35,674 - regression_class_books_logistic - INFO - Model 2
2024-03-13 20:15:35,717 - regression_class_books_logistic - INFO - Model 3
2024-03-13 20:15:35,752 - regression_class_books_logistic - INFO - Model 4
2024-03-13 20:15:35,776 - regression_class_books_logistic - INFO - Model 5
2024-03-13 20:15:35,874 - regression_class_books_logistic - INFO - Model 6
2024-03-13 20:15:35,953 - regression_class_books_logistic - INFO - Model 7
2024-03-13 20:15:36,198 - regression_class_books_logistic - INFO - Model 8
2024-03-13 20:25:17,671 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:25:17,679 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:25:17,683 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:25:17,683 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

